README.md

```diff
@@ -32,9 +32,11 @@
 
 # 📖 [Leaderboard] <a name="about-project"></a>
 
```

Comment:
- [ ] Kindly fix the link in the live link section by using the deployed link or removing the section and its link in the header completely.

- [ ] In your readme file, kindly remove one of the duplicate Future Features section heading.

  ![image](https://github.com/aradradev/leaderboard-app/assets/81468675/5489561c-8649-4761-8482-17211205ae62)